include/IECoreScene/Primitive.inl

```diff
@@ -38,6 +38,75 @@
 namespace IECoreScene
 {
 
+template<typename T>
+std::unique_ptr< PrimitiveVariable::IndexedView<T> >
+Primitive::variableIndexedView( const std::string &name, PrimitiveVariable::Interpolation requiredInterpolation, bool throwIfInvalid )
+{
+	PrimitiveVariableMap::const_iterator it = variables.find( name );
+	if( it == variables.end() )
+	{
+		if ( throwIfInvalid )
+		{
+			throw IECore::InvalidArgumentException( boost::str( boost::format( "Primitive::variableIndexedView - No primvar named '%1%' found" ) % name ) );
+		}
+		else
+		{
+			return nullptr;
+		}
+	}
+
+	if (!it->second.interpolation == requiredInterpolation)
```

Comment:
Should this be `it->second.interpolation != requiredInterpolation`?

Reply:
Gah! There was a leading **!**  :man_facepalming: